ci/release_tidb_operator_binary_and_image.groovy

```diff
@@ -26,11 +26,17 @@ def call(BUILD_BRANCH, RELEASE_TAG, CREDENTIALS_ID, CHART_ITEMS) {
                     images.each {
                         stage("Build and push ${it} image") {
                             withDockerServer([uri: "${env.DOCKER_HOST}"]) {
-                                docker.build("pingcap/${it}:${RELEASE_TAG}", "images/${it}").push()
+                                sh """
+                                docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
Privileged container can be run in Jenkinsfile? It's a critical security issue...

Reply:
I also copied it from other articles. Let't me try

Reply:
Maybe we need to use the image with a specific version and the version is configurable. Also, we use the image from our registry